src/pudl/etl/__init__.py

```diff
@@ -37,6 +37,12 @@
     *load_assets_from_modules([glue_assets], group_name="glue"),
     *load_assets_from_modules([pudl.extract.eia860], group_name="eia860_raw_assets"),
     *load_assets_from_modules([pudl.extract.eia923], group_name="eia923_raw_assets"),
+    *load_assets_from_modules(
+        [pudl.transform.eia860], group_name="pre_harvested_eia860_assets"
+    ),
+    *load_assets_from_modules(
+        [pudl.transform.eia923], group_name="pre_harvested_eia923_assets"
```

Comment:
Why do some of the `group_name`s take the form `{data_source}_{etl_step}_assets` (`eia860_raw_assets`) while others take the form `{etl_step}_{data_source}_assets` (`pre_harvested_eia923_assets`)?